### ISSUE TITLE
Request: wrote a message if there a no issues

### ISSUE THREAD

**Issue by LukeWCS** [squizlabs/PHP_CodeSniffer#2735]
Hi

I use PHPCS in a WAMP environment and with several scripts of my own. It would be very useful if PHPCS would give a message on 0 errors, something like "No issues found" or something similar.

**Comment by gsherwood** [comment]
PHPCS has always been silent by design. Changing that now would potentially break custom scripts and CI processes.

If you'd like some content to be displayed, maybe the `-p` progress option might help.

Or if you want a lot more output, including confirmation of what files were processed and that there were no errors found in each, then the `-v` command line argument might be worth trying.

**Comment by LukeWCS** [comment]
Hi gsherwood, thanks for your answer.

> PHPCS has always been silent by design. Changing that now would potentially break custom scripts and CI processes.

Okay, then this would be a bad idea, understood.

The problem right now is that there are 2 scenarios that are indistinguishable:

1) PHPCS was executed successfully and there were no errors.
2) PHPCS was not performed, for whatever reason.

In both cases, the output in the log file would simply be nothing.

About `-p` and `-v`: After using `php phpcs.phar -h` and seeing these switches, I already experimented with it. However, the problem with `-p` is that this switch is designed for direct viewing, not for log files. And the problem with `-v` would be that then all the files that were tested were listed. And that would be a hindrance in my own scripting environment. I'm only interested in the files that have errors after running PHPCS. I'm just running PHPCS inside of scripts with other tools and then looking at the shared log in which all the tools have written. This log is also read by other users.

At the moment I'm actually using the `-p` switch, because that's how I know if PHPCS was actually executed.

A small suggestion: what do you think about an additional switch named `-v0`, or something similar? This would not affect other scripts, as they do not use this switch. But I can also live with `-p`. ^^

**Comment by gsherwood** [comment]
> A small suggestion: what do you think about an additional switch named `-v0`, or something similar?

A new command line argument could be added to output a success message, but you could probably also do it easily enough by checking the script's return value as well, so I'm not sure how valuable it is.

I'll leave this open to see if there is more support out there for this feature.

**Comment by LukeWCS** [comment]
The return value was also my first approach to solve the problem, but remember: I use PHPCS in a WAMP environment, so Windows. That means I use Batch to control PHP scripts. I've been more involved with PHP since 2018, and usually only with phpBB extensions. How can I determine the return value of a PHP script in this case? If I knew that, then of course I could trigger a "No issues found" message myself.

**Comment by jrfnl** [comment]
@LukeWCS In Windows batch, you can get access to the exit code via the `%errorlevel%` variable directly after running the script.

```bash
ECHO Exit code is %errorlevel%
```

Hope that helps.

**Comment by LukeWCS** [comment]
@jrfnl 

Thanks for you answer.

In this case %errorlevel% can't be used, because this would be the return code from php.exe, not from the PHP script. And I need to know what exactly is the result of the script. But thanks anyways.

@gsherwood 

I found a workaround. Now I use the switch `--report-file` instead of redirecting the output into a log file. So I can now check by batch script, if 1) this report file exists and 2) if the file size is exactly 2 bytes (CRLF). If both conditions apply, then I know that PHPCS was executed successfully and that no errors were found. This is not elegant, but it serves the purpose.
